src/platform/site-wide/banners/components/Banners/index.js

```diff
@@ -20,7 +20,7 @@ export const Banners = ({
       content={homepageBannerContent}
       localStorage={localStorage}
       recordEvent={recordEvent}
-      showClose={!environment.isProduction()}
+      showClose
```

Comment:
Should L8 be removed now that `environment` isn't referenced?

Reply:
Ah good catch! Thanks! 🔥